docs/_tutorials/read-only-json-api.md

```diff
@@ -0,0 +1,354 @@
+---
+title: Read-Only JSON API
+author: izdwuut
+date: 2020-12-23 10:04:00 +0100
+plugin_disclaimer: true
+---
+In this tutorial, you'll create a kind of a [REST](https://restfulapi.net/) [API](https://en.wikipedia.org/wiki/API)
+serving static [JSON](https://www.json.org/json-en.html) files generated from Markdown posts in your Jekyll site.
+
+JSON is a popular cross-language method of transferring data on the web, without all the presentation layer of HTML.
+The files you output will be similar to a REST API in a server-side language like Ruby, Python or Node. You'll be able
```

Comment:
"The files you output will be similar to a REST API in a server side language like Ruby, Python, or Node."